static/app/views/dashboards/detail.tsx

```diff
@@ -928,7 +928,12 @@ class DashboardDetail extends Component<Props, State> {
                                   this.isEditingDashboard
                                 }
                                 isPreview={this.isPreview}
-                                onDashboardFilterChange={this.handleChangeFilter}
+                                onDashboardFilterChange={() => {
+                                  this.handleChangeFilter;
```

Comment:
🤔 I think this should be
```suggestion
                                onDashboardFilterChange={(activeFilters) => {
                                  this.handleChangeFilter(activeFilters);
```

Reply:
I think the test failures are related to this

Reply:
changed some lines so that I now track the event in the `handleChangeFilter` function for this case